assets/js/base/components/payment-methods/test/payment-methods.js

```diff
@@ -61,51 +33,4 @@ describe( 'PaymentMethods', () => {
 			expect( noPaymentMethods.length ).toBeGreaterThanOrEqual( 1 );
 		} );
 	} );
-
-	test( 'should hide/show PaymentMethodOptions when a saved payment method is checked/unchecked', async () => {
-		registerMockPaymentMethods();
-		render(
```

Comment:
I appreciate why these tests are removed because behaviour has changed, but we should make sure the new behaviour is tested.

Reply:
Would still like to make sure we have test coverage on the new ui/ux - is this something planned for a followup?

Reply:
I want to have it with this PR.